lib/uplink/project.go

```diff
@@ -202,3 +207,20 @@ func (p *Project) OpenBucket(ctx context.Context, bucketName string, access *Enc
 func (p *Project) Close() error {
 	return nil
 }
+
+// CheckBucketAttribution Checks the bucket attribution
+func (p *Project) CheckBucketAttribution(ctx context.Context, bucketName string) (bool, error) {
+	partnerID, err := uuid.Parse(p.uplinkCfg.Volatile.PartnerID)
```

Comment:
if there isn't a partner id set, we should just return out with no error

Reply:
Added the logic to handle the not set partnerID and just log the info and return.